### PR TITLE
File Added: JavaScript Assignments 1 & 2

### PR DESCRIPTION
**Pull request by vijaya1811**

### CODE REVIEW

File: vijaya/JavaScript/assignment1/swap.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<!doctype html>
+<html>
+
+<head>
+    <style>
```
Comment (shubhamdalal49) [inline vijaya/JavaScript/assignment1/swap.html]: remove this extra line and check indentation error

File: vijaya/JavaScript/assignment1/swap.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            b = Number(document.getElementById("second").value);
+            a=a+b;
+            b=a-b;
+            a=a-b;
+            document.getElementById("answer1").value = a;
+            document.getElementById("answer2").value = b;
+        }
+    </script>
+</head>
+<body>
```
Comment (shubhamdalal49) [inline vijaya/JavaScript/assignment1/swap.html]: check indentation

File: vijaya/JavaScript/assignment2/student.html [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+                } else if (per >= 76 && per <= 100) {
+                    grade = "A";
+                } else {
+                    grade = "Invalid or Failed";
+                }
+                document.getElementById("demo").innerHTML =
+                    "Your Total Marks : " + total + "<br><br>Your Percentage : " + per + "<br><br>Your Grade : " + grade;
+                // document.write("Your Total Marks : "+total);
+                // document.write("<br>Your Percentage : "+per);
+                // document.write("<br>Your Grade : "+grade);
```
Comment (shubhamdalal49) [inline vijaya/JavaScript/assignment2/student.html]: remove this unnecessary comments

**Review by shubhamdalal49 — CHANGES_REQUESTED** [review]

**Review by Pawan1005 — CHANGES_REQUESTED** [review]
make changes asap

**Review by nileshp1122 — APPROVED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Review by shubhamdalal49 — APPROVED** [review]

**Review by urmilagagare — APPROVED** [review]

**Pull request merged**